docs/immortal-empires/slaanesh.md

```diff
@@ -0,0 +1,32 @@
+# Slaanesh 
+
+## N'Kari - The Arch-Temptor
+
+* **Route I - The Great Game**
+  * Though Slaanesh is presently the least of the four great Ruinous Powers, his brothers secretly fear him, for he feeds on humanity's darkest urges, a source of power that they cannot hope to match. Let the lands of Ulthuan be submerged beneath a writhing, twitching blanket of sin.
+  * Achieve the highest corruption in a minimum of 100 regions
+  * Occupy provinces: Caledor, Tiranoc, Nagarythe, Chrace, Cothique, Northern Yvresse, Southern Yvresse, Eataine, Ellyrion, Avelorn, Saphery.
+  * **Reward**:
+    * _Slaanesh Ascendant_: Allegiance points gained for all factions: +25%, tribute from vassals +100%, Devotees per turn +40
+Alternatives: could also buff things like the Gift of Slaanesh itself, which currently gives +5 devotees and +3 seductive influence per turn PER character's faction, so I stayed away from it because it's harder to control balance-wise
+_Reward flavour text:_ The Dark Prince's power waxes so strongly that it can no longer be contained within the Realms of Chaos. Storms howl outwards from the polar gates, driving his servants onwards to commit countless unspeakable acts in his name.

```

Comment:
Reward flavour text should not be in this file. I am ok with using it during development, but it has to be out before release

Reply:
We need somewhere to discuss everything and see everything before release.

Reply:
Yeah keep it in during design, it can be removed on publish.